README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+Creating tags that correspond with releases and release candidates in [`ansible/ansible`](https://github.com/ansible/ansible)
+
+First, install the additional tagging dependencies
```

Comment:
```suggestion
First, install the additional tagging dependencies as follows:
```

Reply:
There needs to be some kind of ending punctuation for the sentence. Because this is a "lead in" or introductory sentence, I tend to use "as follows:" but you could reword it as you see fit.